File: Лаба2/Задание 3 вариант 3/main.py
inputs = input().split()
n = int(inputs[0])

count = 0
for i in range(1, n + 1):
    g = int(inputs[i])
    number = g
    while number > 0:
        h = number % 10
        number //= 10
        if h % 3 == 0:
            count += h

print(count)

File: Лаба2/Задание 3 вариант 3/main.swift
var count = 0

if let input = readLine() {
    let inputs = input.split(separator: " ")
    if let n = Int(inputs[0]) {
        var remainingInputs = inputs.dropFirst()
        for _ in 0..<n {
            if let k = Int(remainingInputs.removeFirst()) {
                var number = k
                while number > 0 {
                    let h = number % 10
                    number /= 10
                    if h % 3 == 0 {
                        count += h
                    }
                }
            }
        }
    }
}

print(count)
